web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -533,7 +573,12 @@ export default class RunComparison extends ReactJS.Component {
                 <div style={{ margin: 16 }}>
                   <Spin spinning={this.state.loading}>
                     {Object.keys(clusteredGraphData).map(cluster => (
-                      <div>
+                      <div id={'cluster' + cluster}>
+                        <input
+                          type="checkbox"
```

Comment:
So will this be displayed as an actual checkbox or will it be displayed as a simple button that can be pressed?

Reply:
To select the custom divisions, I am planning to have checkbox. Once the divisions are selected, we can download the pdf with clicking on share button. 
![screenshot from 2019-01-28 14-54-39](https://user-images.githubusercontent.com/16955978/51826454-1a0b3580-230d-11e9-8a76-255a0a870ece.png)